VMEncryption/volume-notif-svc/azure-diskencryption-vol-notif.service

```diff
@@ -15,7 +15,7 @@ WorkingDirectory=WorkingDirectory
 ExecStart=WorkingDirectory/ade-volume-notif-svc -d
 #ExecStop=/bin/bash -c '$$(which kill) -15 $MAINPID'
 Delegate=yes
-MemoryMax=50MB
+#MemoryMax=50M
```

Comment:
Let's put a comment that the workingset of ADE is added to the service because it is run a child process. Hence, we don't want to add a limit here.